juniper_official/routing/collect-rsvp-stats.rule

```diff
@@ -0,0 +1,200 @@
+/*
+ * Collect RSVP LSP traffic statistics for pathfinder dynamic topology.
+ *
+ * Req 60: https://paragon-automation.atlassian.net/browse/REQ-60
+ *
+ * One input control detection
+ *
+ *   1) input-lsp-name, is a regular expression that matches the lsp name
+ *      that you would like to monitor.  By default it's '.*', which matches
+ *      all lsps. Use something like 'Silver*' to match only lsp names starting
+ *      with 'Silver'. 
+ *
+ */
+ healthbot {
+    topic routing.mpls {
+        rule collect-rsvp-stats {
+            keys lsp-name;
+            description "Collects LSP stats periodically";
+            sensor lsp {
+                open-config {
+                    sensor-name /junos/services/label-switched-path/usage/;
+                    frequency 60s;
+                }
+            }
+            field lsp-name {
+                sensor lsp {
+                    where "/mpls/lsps/constrained-path/tunnels/tunnel/@name =~ /{{input-lsp-name}}/";
+                    path "/mpls/lsps/constrained-path/tunnels/tunnel/@name";
+                }
+                type string;
+                description "LSP name to be monitored";
+            }
+            field lsp-stats-bps {
+                formula {
+                    rate-of-change {
+                        field-name "$lsp-stats-bytes";
+                        multiplication-factor 8;
+                    }
+                }
+                type integer;
+            }
+            field lsp-stats-bytes {
+                sensor lsp {
+                    path /mpls/lsps/constrained-path/tunnels/tunnel/state/counters/bytes;
+                }
+                type integer;
+                description "LSP bytes counter";
+            }
+            field lsp-stats-counter {
+                sensor lsp {
+                    path /mpls/lsps/constrained-path/tunnels/tunnel/state/counters/name;
```

Comment:
What is the significance of this? Would there be different counter names? Any documentation? or what values can come here?

Reply:
Could not find these path's in telemetry explorer

Reply:
As per mail from Reinhard lisp-name also should be in the key

Reply:
yes, lsp-name is key.

Reply:
Added "lsp-stats-counter" also as the key.